test/snapp_auction.js

```diff
@@ -118,14 +103,32 @@ contract("SnappAuction", async (accounts) => {
     it("Generic sell orders over two batches", async () => {
       const instance = await SnappAuction.new()
       await setupEnvironment(MintableERC20, instance, token_owner, [user_1], 2)
-      await instance.placeSellOrder(0, 1, 1, 1, { from: user_1 })
+      const order = encodeOrder(0, 1, 1, 1)
+      await instance.placeSellOrders(order, { from: user_1 })
 
       await waitForNSeconds(181)
-      await instance.placeSellOrder(0, 1, 1, 1, { from: user_1 })
+      await instance.placeSellOrders(order, { from: user_1 })
 
       const auctionIndex = (await instance.auctionIndex.call()).toNumber()
       assert.equal(auctionIndex, 1)
     })
+
+    it("Generic Multi order (2)", async () => {
+      const instance = await SnappAuction.new()
+      await setupEnvironment(MintableERC20, instance, token_owner, [user_1], 2)
+      const order1 = encodeOrder(0, 1, 3, 4)
+      const order2 = encodeOrder(1, 0, 1,   123456789)
+      // const order2 = encodeOrder(1, 0, 1,   100000000000000009) --> shows 100000000000000020
```

Comment:
Do we really want to commit this?

Reply:
I guess not, but I really want to make it clear that this is a pending topic of concern. I will create an issue for this instead.